docs/content/doc/advanced/specific-variables.en-us.md

```diff
@@ -25,43 +25,45 @@ GITEA_CUSTOM=/home/gitea/custom ./gitea web
 
 ## From Go language
 
-As Gitea is written in Go, it uses some Go variables as:
+As Gitea is written in Go, it uses some Go variables, such as:
 
   * `GOOS`
   * `GOARCH`
-  * `GOPATH`
+  * [`GOPATH`](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable)
 
-For `GOPATH`, check [official documentation about GOPATH environment variable](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable).
-
-For others, check [official documentation about variables used when it runs the generator](https://golang.org/cmd/go/#hdr-Generate_Go_files_by_processing_source).
+For documentation about each of the variables available, refer to the
+[official documentation](https://golang.org/cmd/go/#hdr-Generate_Go_files_by_processing_source).
```

Comment:
I did not point it out as I should, but I think referencing https://golang.org/cmd/go/#hdr-Environment_variables instead is better